models/public/hbonet-0.25/hbonet-0.25.md

```diff
@@ -0,0 +1,62 @@
+# hbonet-0.25
+
+## Use Case and High-Level Description
+
+The `hbonet-0.25` model is one of the classification models from https://github.com/d-li14/HBONet with `width_mult=0.25`
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 0.037         |
+| MParams           |               |
```

Comment:
Don't forget to add this value (also to index)

Reply:
What is BKM in case of missed metric in the origin paper?

Reply:
You could use metrics from IRs

Reply:
Is there a method which can return number of parameters for IR? That means I should generate a model with unfused layers first.

Reply:
You can use `model_analyzer` tool from OpenVINO package.

Reply:
Thank you! Added the metrics using `--disable_fusing` MO flag.